openedx/core/djangoapps/content_tagging/rest_api/v1/serializers.py

```diff
@@ -94,3 +95,19 @@ class Meta:
         model = TaxonomySerializer.Meta.model
         fields = TaxonomySerializer.Meta.fields + ["orgs", "all_orgs"]
         read_only_fields = ["orgs", "all_orgs"]
+
+
+class ObjectTagOrgMinimalSerializer(ObjectTagMinimalSerializer):
+    """
+    Serializer for Object Tags
```

Comment:
Can you update the docstring to explain why/how this is different from the base class? (It overrides permissions so users can't delete copied tags?)

Reply:
Also I don't see why we have "Org" in the name when the code has nothing to do with organizations.

Reply:
Updated: https://github.com/openedx/edx-platform/pull/35596/commits/d9bba2c1b9f6206074da6fce39b714228b738801